src/helpers/blood-pressure-data-providers/survey-blood-pressure-data-provider.ts

```diff
@@ -0,0 +1,81 @@
+import MyDataHelps, { Guid, SurveyAnswer, SurveyAnswersPage, SurveyAnswersQuery } from "@careevolution/mydatahelps-js";
+import { BloodPressureDataPoint } from "./query-blood-pressure";
+import { format, parseISO, startOfDay, toDate } from "date-fns";
+
+export interface BloodPressureDataParameters {
+    surveyName: string,
+    dateOfResultId? : string,
```

Comment:
These should be suffixed "ResultIdentifier" for clarity, e.g. "systolicResultIdentifier"

Reply:
Also doesn't make a ton of sense that this is called "dateOfResultIdentifier", I would shorten to "dateResultIdentifier"